parquet-hadoop/src/main/java/org/apache/parquet/format/converter/ParquetMetadataConverter.java

```diff
@@ -81,6 +82,16 @@
 
   private static final Log LOG = Log.getLog(ParquetMetadataConverter.class);
 
+  private final boolean useSignedStringMinMax;
+
+  public ParquetMetadataConverter() {
+    this.useSignedStringMinMax = false;
+  }
+
+  public ParquetMetadataConverter(Configuration conf) {
+    this.useSignedStringMinMax = conf.getBoolean("parquet.strings.use-signed-order", false);
```

Comment:
I changed the property to `parquet.strings.signed-min-max.enabled` and moved it to `ParquetInputFormat`. The different name is because it only affects strings.

I initially changed the parameter to a boolean as you suggested, but when we move to a better stats representation and pass the order used to produce min/max along with stats, we won't need this any more. It will be a useless part of the public API, but we may have another situation where we need a Configuration in the future, so I think it makes more sense to go with this version.